.gitignore

```diff
@@ -1 +1,26 @@
-test.js
```

Comment:
dont know what was this before

Reply:
i guess its dummy file. nothing to worry about this